distribution/docs/src/main/resources/content/_configuring/configuring-fanout-proxy.adoc

```diff
@@ -20,4 +20,4 @@ All queries are enterprise queries and no catalog provider needs to be configure
 ${branding} is now operating as a fanout proxy.
 Only queries and resource retrieval requests will be allowed.
 All queries will be federated.
-Create, update, and delete requests will throw an `UnsupportedOperationException`, even if a Catalog Provider was configured prior to the reconfiguration to fanout.
+Create, update, and delete requests will not be allowed, even if a Catalog Provider was configured prior to the reconfiguration to fanout.
```

Comment:
✏️ prior to the reconfiguration `as a` fanout.

Reply:
fixed.